config/packages/test/validator.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
```

Comment:
This file has created with composer, but not sure it's needed for now.

Reply:
It's a recipe file so we can leave it  https://github.com/symfony/recipes/tree/master/symfony/validator/4.3